src/olympia/lib/settings_base.py

```diff
@@ -1309,7 +1305,6 @@ def get_db_config(environ_var, atomic_requests=True):
     PROD_STATIC_URL,
 )
 CSP_STYLE_SRC = (
-    "'self'",
```

Comment:
don't we need this for admin pages that have some django css?

Reply:
django admin css should be served from `STATIC_URL` (`https://a.m.o./static/`)